profiles/base.yaml

```diff
@@ -61,6 +61,7 @@ dance_actions:
 - circle
 - bob
 - weave
+use_tactics_combos: false
```

Comment:
Either or, I didn't see the setting while editing it directly on github. I did see it later though!